assets/css/_search_form.scss

```diff
@@ -1,95 +1,97 @@
-.c-search-form__row {
+.c-search-form {
   display: flex;
+  flex-flow: column nowrap;
+  gap: 1rem;
+}
 
-  &:not(:last-child) {
-    margin-bottom: 1rem;
-  }
+.c-search-form__search-control {
+  // Make control area match designs
+  height: 2.25rem;
 }
 
-.c-search-form__text {
+.c-search-form__search-input-container {
+  // Layout floating `input-controls` over the `search-input-container`
   position: relative;
-  flex: 1 1 auto;
+
+  // Use bootstrap control as base for now
+  @extend .form-control;
```

Comment:
**question:** what are the relative merits of extending the `form-control` style versus customizing it via the available Sass variables? It seems like some of this styling is definitely more specific (mainly leaving that extra space for the controls), but other parts about the colors and focus state look like they could maybe be accommodated via the Sass method.

Reply:
> versus customizing it via the available Sass variables
I didn't want to assume we'd be going with bootstrap as a design system, so I didn't want to spend time customizing bootstrap, and instead try and be as close to the way we're currently doing CSS within Skate.

My thought process for extending `form-control` was
- it should give us an easier transistion to bootstrap for this component if we use bootstrap as our design system.
- it should allow us to reference an existing component+style if we're moving to a different design system.

I contemplated inlining bootstraps styles or pruning it down, but I figured the dev time wasn't worth it since this worked.

I'm open to configuring this via SASS/CSS variables if you think it's worth it, I just didn't want to assume that this is what _all_ our `form-controls` should follow in terms of color/focus.

Reply:
Makes sense! I wasn't sure exactly how all of your conversations with design had panned out, and I understand not wanting to make premature decisions about what parts of this styling will be universal.